src/models/ApiMapper.php

```diff
@@ -36,32 +38,42 @@ public function transformResults($results, $verbose)
             $entry = array();
             foreach ($fields as $key => $value) {
                 // special handling for dates
-                if (substr($key, -5) == '_date' && !empty($row[$value])) {
-                    if ($row['event_tz_place'] != '' && $row['event_tz_cont'] != '') {
+                if (substr($key, - 5) == '_date' && ! empty($row[ $value ])) {
+                    if ($row['event_tz_place'] != ''
+                        && $row['event_tz_cont'] != ''
+                    ) {
                         $tz = $row['event_tz_cont'] . '/' . $row['event_tz_place'];
                     } else {
                         $tz = 'UTC';
                     }
-                    $entry[$key] = Timezone::formattedEventDatetimeFromUnixtime($row[$value], $tz, 'c');
+                    $entry[ $key ] =
+                        Timezone::formattedEventDatetimeFromUnixtime(
+                            $row[ $value ],
+                            $tz,
+                            'c'
+                        );
                 } else {
-                    $entry[$key] = array_key_exists($value, $row) ? $row[$value] : null;
+                    $entry[ $key ] = array_key_exists($value, $row) ?
```

Comment:
A ternary operator on two lines is just wrong! Can you swap to an if statement please.